plat/kvm/x86/setup.c

```diff
@@ -69,7 +256,32 @@ static void __noreturn _ukplat_entry2(void)
 
 	ukplat_lcpu_halt();
 }
+#ifdef CONFIG_RUNTIME_ASLR
+__u32 _gen_seed32(){
+	__u32 low,high;
+	__asm__ __volatile__ ("rdtsc" : "=a" (low), "=d" (high));
+	return ((unsigned long long)high << 32) | low;
```

Comment:
```suggestion
	return ((__u64)high << 32) | low;
```
Change the function signature to return a `__u64` too.